packages/portfolio/src/helpers/create-cached-unknown-token-info.ts

```diff
@@ -0,0 +1,16 @@
+import {App, Portfolio} from '@yoroi/types'
+import {freeze} from 'immer'
+
+import {createUnknownTokenInfo} from './create-unknown-token-info'
+
+export const createCachedUnknownTokenInfo = (
```

Comment:
api fails won't block the wallet anymore, tokens will be created as unknown and will be stale by default, so it will refresh them every chance until get something from API.